Makefile

```
@@ -204,3 +204,29 @@ migrate: migrate-lms migrate-cms
 # Part of https://github.com/openedx/wg-developer-experience/issues/136
 ubuntu-requirements: ## Install ubuntu 22.04 system packages needed for `pip install` to work on ubuntu.
 	sudo apt install libmysqlclient-dev libxmlsec1-dev
+
+eslint:
```

Please add comments to each of these, using `##`, so that they show up in the auto-generated `make help` message. Here's an example.
```suggestion
eslint:  ## check javascript for quality issues
```